modules/adopter-registration-impl/src/main/java/org/opencastproject/adopter/statistic/ScheduledDataCollector.java

```diff
@@ -100,6 +121,9 @@ public class ScheduledDataCollector extends TimerTask {
   /** The Opencast version this is running in */
   private String version;
 
+  /** The timer for shutdown uses */
+  private Timer t;
```

Comment:
Maybe more than a single character?
```suggestion
  private Timer timer;
```